### DISCUSSION TITLE
Unable to find sample data to try with tutorial

### DISCUSSION THREAD

**Discussion by mthimma** [brianhie/scanorama#91] [Q&A]
Dear BrianHie,

could you please provide the link for the sample data ./data/results/scanpy_dr_covid.h5ad' to try with your tutorial?

Best,
M

**Comment by brianhie** [comment]
I’m not sure what file you are referring to, which tutorial lists that as an input file?

**Reply by mthimma** [reply]
Hi Hie,
I am trying to integrate pbmc datasets as mentioned in your tutorial.
https://nbisweden.github.io/workshop-scRNAseq/labs/compiled/scanpy/scanpy_03_integration.html

There in the box [2] has importing data, but not able to get hold of your sample data.
Could you please help me with it?

**Reply by brianhie** [reply]
See here on generating that data: https://nbisweden.github.io/workshop-scRNAseq/labs/compiled/scanpy/scanpy_01_qc.html and https://nbisweden.github.io/workshop-scRNAseq/labs/compiled/scanpy/scanpy_02_dim_reduction.html
